src/components/Layout/PageLayout/index.tsx

```diff
@@ -1,12 +1,23 @@
 import React from "react";
 import { Box, CssBaseline, Container } from "@mui/material";
-import { pageLayoutStyles } from "./pageLayoutStyles";
+import { Style } from "shared/theme";
 
 interface IProps {
   header: React.ReactNode;
   children: React.ReactNode;
 }
 
+const pageLayoutStyles: Style = {
+  containerWrapper: {
+    height: "100vh",
```

Comment:
Why at that level? prefer height to the document itself instead

Reply:
Fixed